.bundle/config

```diff
@@ -1,2 +1,3 @@
 ---
 BUNDLE_FROZEN: "1"
+BUNDLE_WITHOUT: "omnibus_package:test:pry:integration:docgen:maintenance:travis:aix:bsd:linux:mac_os_x:solaris:windows:development"
```

Comment:
`rake dependencies:update_gemfile_lock`

but you'll still pull in ohai which will break amazon linux -- working on that.